include/json/value.h

```diff
@@ -230,7 +230,7 @@ class JSON_API Value {
 
     struct StringStorage {
       unsigned policy_: 2;
-      unsigned length_: 30; // 1GB max
+      size_t length_: 30; // 1GB max
```

Comment:
Hi Chris, yes to correctly clean the string on unset it was i'm afraid. I'll put the ifdef around it again like you've suggested elsewhere